### ISSUE TITLE
DateRangeAggregation - "keyed" option support missing

### ISSUE THREAD

**Issue by pmishev** [ongr-io/ElasticsearchDSL#283]
The DateRangeAggregation can't be passed the `keyed` option currently. 
https://www.elastic.co/guide/en/elasticsearch/reference/6.3/search-aggregations-bucket-daterange-aggregation.html#_keyed_response_4

Looks like this has been reported before in #237, but never fixed.

**Comment by saimaz** [comment]
yep, `keyed` is not implemented, would be awesome if you could add a PR to `5.x` branch by introducing this option. It's pretty simple to add though.

**Comment by pmishev** [comment]
Sorry, did it from `master` in a hurry. Hope it's all right
